src/components/CalendarMonthGrid.jsx

```diff
@@ -55,6 +57,7 @@ const defaultProps = {
   onDayTouchTap() {},
   onMonthTransitionEnd() {},
   transform: 'none',
+  daySize: 39,
```

Comment:
if `39` is important, could it be a reused constant, instead of being hardcoded in multiple places?